docs/usage/docs.md

```diff
@@ -2,6 +2,231 @@
 title: Documentation
 ---
 
-This site is built with [Eleventy](https://www.11ty.dev/) and hosted on [Heroku](https://heroku.com).
-All of the pages on the site live in the [docs directory]({{ 'docs' | repo_url }}), and most are
-written in [Markdown](https://www.markdownguide.org/).
+This site is built with [Eleventy] and hosted on [Heroku]. All of the pages on
+the site live in the [docs directory](../..), and most are written in either
+[Markdown] or HTML.
+
+## Eleventy
+There are a couple of things to note about our Eleventy setup:
+
+- Markdown documents are processed first with [Nunjucks](#nunjucks), then
+  with our custom [Markdown pipeline](#remark)
+- We use a handful of custom [template plugins](#template-plugins)
+
+## Markdown
+Whenever possible, please write your documents in [Markdown] rather than HTML.
+The syntax is easier to read and write, and allows us to enhance the content
+programmatically with [remark](#remark).
+
+### Frontmatter
+The term [frontmatter][] (sometimes written as "front matter" or "front
+matter") was popularized by a static site generator called [Jekyll], which
```

Comment:
could we omit the second "frontmatter?" Unless it's supposed to be a different alternate spelling